src/osbuild-manifests/coreos.osbuild.s390x.secex.mpp.yaml

```diff
@@ -0,0 +1,342 @@
+version: '2'
+mpp-vars:
+  filename: $filename
+  ociarchive: $ociarchive
+  osname: $osname
+  buildid: $buildid
+  container_imgref: $container_imgref
+  metal_image_size_mb: $metal_image_size_mb
+  cloud_image_size_mb: $cloud_image_size_mb
+  bios_boot_size_mb: 1
+  ppc_prep_size_mb: 4
+  reserved_part_size_mb: 1
```

Comment:
This is not used.

Reply:
yep, i tried to make `secex.mpp` and  `s390x.mpp` similar. But can drop those vars